recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: conda-forge-build-setup
-  version: 4.4.18
+  version: 4.4.17
```

Comment:
Sorry this is a bit pedantic, but maybe we should bump the version so we don't accidentally try to release 4.4.18 again.

Reply:
Oops. Sorry about that